src/lib/routes.js

```diff
@@ -26,23 +27,39 @@ module.exports = server => {
             next(new server.errors.InternalServerError(err));
           });
       }
-
-      next(new server.errors.BadRequestError(`Make sure the date format is '${dateFormat}' (time is optional)`));
+      next(new server.errors.BadRequestError(`Make sure the date format is "${dateFormat}" (time is optional)`));
     } else {
-      next(new server.errors.BadRequestError('Must provide a \'from\' date parameter, and optionally a \'to\' date'));
+      next(new server.errors.BadRequestError('Must provide a "from" date parameter, and optionally a "to" date'));
     }
   });
 
+  server.get('/dashboard', (req, res) => {
+    dashboardModel()
+      .then(props => res.render(LevDashboard, props))
+      .catch(err => server.log.error(err));
```

Comment:
Possibly we should explicitly call `next()` here.

Or maybe call next with err so it is picked up by the error handler.

If we are going to do logging here (should we, shouldn't the error handler do it for us) we should use `req.log` instead.

Reply:
Refactored